canonical-web-and-design/ubuntu.com
# Telco bubble

## Done

- Updates /telco pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against found in the [brief](https://docs.google.com/document/d/1oZV1VdLUPl8zt3fTgX-B3j9ksY8Esq4PWnEAI_cwOmg/edit#)
- Make sure navigation and contact form works

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4185
